nautobot/core/ui/object_detail.py

```diff
@@ -685,10 +692,23 @@ def render_body_content(self, context):
         result = format_html("")
         for key, value in data.items():
             key_display = self.render_key(key, value, context)
-            value_display = self.render_value(key, value, context)
-            if value_display:
-                # TODO: add a copy button on hover to all display items
-                result += format_html("<tr><td>{key}</td><td>{value}</td></tr>", key=key_display, value=value_display)
+            if value_display := self.render_value(key, value, context):
+                if value in [None, ""]:
+                    value_tag = value_display
+                else:
+                    value_tag = format_html(
+                        """
+                            <span class="hover_copy">
+                                <span id="{key}_copy">{value}</span>
```

Comment:
Possibly something like `value_{key}` might be clearer as an `id` string?

But also need to consider that keys may not be globally unique in the page, for example the "termination A" and "termination Z" panels in the Circuit detail view. Might need to *also* include the panel's `id` as a part of this string?

Do we need to furthermore `slugify` the `key` or anything like that to have correct handling of whitespace in keys, or is that handled just fine as-is?

Reply:
I think there was at least one case in the circuits view where I was passing a display label as the `key` because the automatic rendering didn't handle capitalization correctly. Not sure this is a safe assumption to make.